kernel/build/config-arm64

```diff
@@ -5777,7 +5784,428 @@ CONFIG_LOGO_LINUX_CLUT224=y
 # end of Graphics support
 
 # CONFIG_DRM_ACCEL is not set
-# CONFIG_SOUND is not set
+CONFIG_SOUND=y
```

Comment:
can we make all this modules?

Reply:
Probably, I will try that out.

Reply:
@frezbo I changed it to a minimal working config and set it to build modules where possible.